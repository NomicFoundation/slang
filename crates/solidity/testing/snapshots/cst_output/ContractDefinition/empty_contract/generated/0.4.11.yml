# This file is generated automatically by infrastructure scripts. Please don't edit by hand.

Source: >
  1  │ contract Sample {}                                                               │ 0..18

Errors: []

Tree:
  - ContractDefinition (Rule): # 0..19 "contract Sample {}\n"
      - ContractKeyword (Token): "contract" # 0..8
      - Identifier (Token): "Sample" # 9..15
      - OpenBrace (Token): "{" # 16..17
      - ContractBodyElements (Rule): [] # 17..17
      - CloseBrace (Token): "}" # 17..18
