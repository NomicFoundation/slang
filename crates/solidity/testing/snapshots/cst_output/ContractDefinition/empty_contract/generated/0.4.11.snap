# This file is generated automatically by infrastructure scripts. Please don't edit by hand.

Errors: 0

█ Rule(ContractDefinition) - (0..18)     │
├──► Token(Contract) - (0..8)            │ contract
├──► Token(Identifier) - (9..15)         │ Sample
└──█ Group - (16..18)                    │
   ├──► Token(OpenBrace) - (16..17)      │ {
   └──► Token(CloseBrace) - (17..18)     │ }
